lib/src/main/java/graphql/nadel/NextgenEngine.kt

```diff
@@ -117,14 +120,17 @@ internal class NextgenEngine(
                 )
             }
 
+            val incrementalResultSupport = NadelIncrementalResultSupport()
```

Comment:
New defer support class that gets stored in `NadelExecutionContext` for transforms etc to access.